CartMgmt/Cart/cart.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --white-color: #fff;
+  --grey-color: #808080;
+  --peach-color:rgb(241, 203, 219);
+
+}
+.navigation {
+  margin: 0rem;
+  width: 100%;
+  background-color: var(--peach-color);
+}
+
+.icon-heart,
+.icon-cart {
+  font-size: 3rem;
+}
+
+.cart-item {
+  flex: 50;
+}
+
+.cart-price {
+  margin: 5rem;
+  flex: 50;
+}
+
+.price-container {
+  align-items: center;
+  background-color: var(--peach-color);
+  border-radius: 5rem;
+  border-color: antiquewhite;
+  margin-right: 30rem;
+  margin-top: 20rem;
+  padding: 3rem;
+}
+
+.btn-shopping-cart:hover {
+  color: black;
+}
+
+table {
```

Comment:
It's good to use wrapper instead of applying styling on element directly.

Reply:
_ok _